server/tests/models/ticket_spec.js

```diff
@@ -0,0 +1,57 @@
+import chai, { expect, assert } from 'chai';
```

Comment:
We should probably stay consistent with `asssert` vs. `expect`. We've used `expect` in the other tests and it is used more in bdd. `expect(theThingImTesting).to.be.equal(theThingIWantMyTestToEqual)`.